devices/HumanStateProvider/HumanStateProvider.cpp

```diff
@@ -1517,6 +1524,46 @@ bool HumanStateProvider::impl::applyRpcCommand()
         computeSecondaryCalibrationRotationsForChain(jointZeroIndices, iDynTree::Transform::Identity(), linkToCalibrateIndices, "");
         break;
     }
+    case rpcCommand::calibrateAllWorldYaw: {
+        // Select all the links and the joints
+        linkToCalibrateIndices.resize(kinDynComputations->getNrOfLinks());
+        std::iota(linkToCalibrateIndices.begin(), linkToCalibrateIndices.end(), 0);
+
+        // add all the joints of the model to [jointZeroIndices]
+        jointZeroIndices.resize(kinDynComputations->getNrOfDegreesOfFreedom());
+        std::iota(jointZeroIndices.begin(), jointZeroIndices.end(), 0);
+
+        // initialize vectors
+        iDynTree::VectorDynSize jointPos(jointConfigurationSolution);
+        iDynTree::VectorDynSize jointVel(jointVelocitiesSolution);
+        jointVel.zero();
```

Comment:
no need to initialize this with a copy constructor, since it's anyways zeroed. We may initialize using only the size, instead.